src/components/Home/Aditya/Week4/pages/Week3/components/form/Form.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect, useState, useRef } from 'react';
+import { FormWrapper } from './formStyles';
+import { getTodayDate, removeWhitespace } from '../../utils/functions';
+import { Profile } from '../../interfaces/interfaces';
+
+const Form: React.FC = () => {
+  const inputRef = useRef<HTMLInputElement>(null);
+  const [profile, setProfile] = useState<Profile>({
+    name: '',
```

Comment:
move this to a constant initialstate you can utilise it inside handlesubmit function to reset back to this state